develop/dev-guide-gui-navicat.md

```diff
@@ -0,0 +1,152 @@
+
+4. Click **Create password** to create a random password.
+
+    > **Tip:**
+    >
+    > If you have created a password before, you can either use the original password or click **Reset password** to generate a new one.
+
+5. Launch Navicat and click the **Connection** in the upper-left corner. In the selection panel, click **MySQL**  as the database type.
+
+    ![Navicat: add new connection](/media/develop/navicat-add-new-connection.jpg)
+
+6. In the setting panel, configure the following connection parameters:
+
+    - **Connection Name**: give this connection a meaningful name.
+    - **Host**: enter the `host` parameter from the TiDB Cloud connection dialog.
+    - **Port**: enter the `port` parameter from the TiDB Cloud connection dialog.
+    - **Username**: enter the `user` parameter from the TiDB Cloud connection dialog.
+    - **Password**: enter the password of the TiDB Serverless cluster.
+
+    ![Navicat: configure connection general panel for TiDB Serverless](/media/develop/navicat-connection-config-serverless-general.png)
+
+7. Click the **SSL** tab, check **Use SSL**, **Use authentication**, **Verify server certificate against CA**, and enter the `ssl_ca` parameter from the TiDB Cloud connection dialog into the **CA Certificate** input box. **Test Connection** to validate the connection to the TiDB Serverless cluster.
+
+    ![Navicat: configure connection SSL panel for TiDB Serverless](/media/develop/navicat-connection-config-serverless-ssl.png)
+
+8. If the connection test is successful, you can see the **Connection Successful** message. Click **Save** to save the connection configuration.
+
+</div>
+<div label="TiDB Dedicated">
+
+1. Navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page, and then click the name of your target cluster to go to its overview page.
+
+2. Click **Connect** in the upper-right corner. A connection dialog is displayed.
+
+3. Click **Allow Access from Anywhere**.
+
+    For more details about how to obtain the connection string, refer to [TiDB Dedicated standard connection](https://docs.pingcap.com/tidbcloud/connect-via-standard-connection).
+
+4. Launch Navicat and click the **Connection** in the upper-left corner. In the selection panel, click **MySQL**  as the database type.
+
+    ![Navicat: add new connection](/media/develop/navicat-add-new-connection.jpg)
+
+5. In the setting panel, configure the following connection parameters:
+
+    - **Connection Name**: give this connection a meaningful name.
+    - **Host**: enter the `host` parameter from the TiDB Cloud connection dialog.
+    - **Port**: enter the `port` parameter from the TiDB Cloud connection dialog.
+    - **Username**: enter the `user` parameter from the TiDB Cloud connection dialog.
+    - **Password**: enter the password of the TiDB Dedicated cluster.
+
+    ![Navicat: configure connection general panel for TiDB Dedicated](/media/develop/navicat-connection-config-dedicated-general.png)
+
```

Comment:
Shouldn't we include something about SSL/TLS here?

Reply:
Added